api/seeds/test.ts

```diff
@@ -0,0 +1,77 @@
+import bcrypt from 'bcryptjs'
+import { USER_PROFILE } from '@locokit/definitions'
+import { Knex } from 'knex'
+
+export async function seed(knex: Knex): Promise<any> {
+  try {
+    const hashPassword = await bcrypt.hash('locokit', 10)
+
+    const trx = await knex.transaction()
+    /**
+     * remove each workspace
+     */
+    console.log('Removing all workspace dedicated schemas')
```

Comment:
Do we need all this `console.log` ?

Reply:
I prefer, this file is for testing purpose.
When you run tests, you'll see it in your console to check where you are in the execution.
I find this useful indeed.

Reply:
I would have removed the `...ing`, to avoid too much noise but it's ok